express/scripts/scripts.js

```diff
@@ -928,6 +928,11 @@ export function decorateButtons(block = document) {
   const noButtonBlocks = ['template-list', 'icon-list'];
   block.querySelectorAll(':scope a').forEach(($a) => {
     const originalHref = $a.href;
+    if ($a.children.length > 0) {
+      // We can use this to eliminate styling so only text
+      // propagates to buttons.
+        $a.innerHTML = $a.innerHTML.replaceAll('<u>', '').replaceAll('</u>', '');
```

Comment:
```suggestion
      $a.innerHTML = $a.innerHTML.replaceAll('<u>', '').replaceAll('</u>', '');
```
Fix indendation problem